sql/step_a113_alter_table_medicalinventory.sql

```diff
@@ -0,0 +1,12 @@
+-- Step 1: Drop the foreign key constraint
+ALTER TABLE OH_MEDICALDSRINVENTORY 
+DROP CONSTRAINT oh_medicaldsrinventory_ibfk_2;
+
+-- Step 2: Change the data type of the foreign key column
+ALTER TABLE OH_MEDICALDSRINVENTORY 
+MODIFY MINVT_WRD_ID_A CHAR(3);
+
+-- Step 3: Recreate the foreign key constraint
+ALTER TABLE OH_MEDICALDSRINVENTORY 
+ADD CONSTRAINT oh_medicaldsrinventory_ibfk_2
```

Comment:
Let's take the opportunity for renaming the FK with `FK_<THISTABLE>_<REFTABLE>`

And the same we can do for other `oh_medicaldsrinventory_ibfk_*` existing foreign keys

```suggestion
ADD CONSTRAINT FK_MEDICALDSRINVENTORY_WARD
```

Reply:
To avoid duplication we can add `FK_<THISTABLE>_<REFTABLE>_<OTHERMNEMONIC>`

Reply:
> And the same we can do for other oh_medicaldsrinventory_ibfk_* existing foreign keys
> To avoid duplications we can add FK_<THISTABLE>_<REFTABLE>_<OTHERMNEMONIC>

My suggestion was beyond `oh_medicaldsrinventory_ibfk_2`